johnsonandjohnson/Bodiless-JS
# fix(migration-tool): Improve script content transformation

## Changes
* improved the logic to transform content of script html elements that was implemented in order to transform script content with new line for work around an issue with babel. replaced html method with text

## Test Instructions
Test Case 1.
Steps:
1. flatten, build and serve bebe.de
2. open frontpage.

Actual: cookie banner can not be closed
Expected: cookie banner can be closed as on production

Test Case 2.
Steps:
1. flatten, build and serve  biafine.pt
2. open frontpage
3. check link of the image above "Saiba tudo sobre o Biafine"

Actual: link points to /%27+url+%27/
Expected:  link points to /o-que-e-biafine/
